isilon_api.gemspec

```diff
@@ -11,23 +11,23 @@ Gem::Specification.new do |spec|
 
   spec.summary       = %q{Read only API calls for the Isilon OneFS api}
   spec.description   = %q{Read only API calls for the Isilon OneFS api.}
-  spec.homepage      = ""
+  spec.homepage      = "https://github.com/tulibraries/isilon_api"
   spec.license       = "MIT"
 
 
   spec.files         = `git ls-files -z`.split("\x0").reject do |f|
     f.match(%r{^(test|spec|features)/})
   end
 
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.bindir        = "bin"
+  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]\
 
-  spec.add_dependency "faraday"
-
+  #spec.add_dependency "faraday"
+  spec.add_runtime_dependency "faraday", "~> 0"
   spec.add_development_dependency "bundler", "~> 1.13"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
-  spec.add_development_dependency "webmock"
-  spec.add_development_dependency "pry"
+  spec.add_development_dependency "webmock", "~> 0"
```

Comment:
@skng5 This line installed version `0.91` of webmock, which is currently at version `2.x`. It failed because `0.91` didn't have rspec support.